R/link_to_nhd.R

```diff
@@ -39,7 +47,25 @@ link_to_waterbodies = function(lats, lons, ids){
   #TODO: Finish this
   for(i in 1:nrow(to_check)){
     #get nhd layer
-    nhd       = readOGR(to_check[i,'file'])
+    dl_file = ""
+    
+    if(tolower(dataset) == "nhd"){
```

Comment:
Could you put all of these assignments into one if/else that occurs at the beginning of the function? Easier to fix up later and a lot cleaner.